drivers/sensor/adxrs290/adxrs290.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2017 IpTronix S.r.l.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SENSOR_ADXRS290_H__
+#define __SENSOR_ADXRS290_H__
+
+#include <stdint.h>
+#include <device.h>
+
+/* read command */
+#define ADXRS290_READ             0x80
+
+/* Gyroscope */
+#define ADXRS290_ANALOG_ID        0x00
+#define ADXRS290_ANALOG_ID_RETURN 0xAD
+
+#define ADXRS290_MEMS_ID          0x01
+#define ADXRS290_MEMS_ID_RETURN   0x1D
+
+#define ADXRS290_DEV_ID           0x02
+#define ADXRS290_DEV_ID_RETURN    0x92
+
+#define ADXRS290_REV_NUM          0x03
+#define ADXRS290_REV_NUM_RETURN   0x1D
+
+#define ADXRS290_SERIALNUM_START  0x04
+#define ADXRS290_SERIALNUM_END    0x07
+
+/* 16 2-complement's bits */
+#define ADXRS290_GYR_X_L          0x08
+#define ADXRS290_GYR_X_H          0x09
+
+/* 16 2-complement's bits */
+#define ADXRS290_GYR_Y_L          0x0A
+#define ADXRS290_GYR_Y_H          0x0B
+
+/* 12 2-complement's bits */
+#define ADXRS290_TEMP_L           0x0C  /* 7..0 bits */
+#define ADXRS290_TEMP_H           0x0D  /* 11..8 bits */
+#define ADXRS290_TEMP_SCALE_FACT  10.0  /* 10 LSB /centig deg. 0 means 0 deg.*/
```

Comment:
Defined by not used. Also don't see the point in using floating point.

Maybe also define and use a scale factor constant for gyro.